lib/rails_anonymous_controller_testing/railtie.rb

```diff
@@ -60,6 +64,7 @@ def self.controller(base_controller, routes: nil, &block)
           end
 
         anonymous_view_path = _anonymous_view_base_path.join("#{display_name}_#{unique_identifier}")
+        _anonymous_controller_name = controller_name
```

Comment:
This is currently setting a variable of the name `_anonymous_controller_name` instead of using the method you defined you defined in this PR. Adding `self.` to it will fix that.

```suggestion
        self._anonymous_controller_name = controller_name
```